### PR TITLE
Update template 2.13.1

### PR DESCRIPTION
**Pull request by d4straub**
Closes https://github.com/nf-core/ampliseq/issues/712

I had to change `test_iontorrent` & `test_pacbio_its` to not use dada2 with unite but sintax (were cancelled because of too high memory requirements). That meant unfortunately disabling [addsh](https://nf-co.re/ampliseq/2.8.0/parameters#addsh) which is now not tested anywhere, I think.

## PR checklist

- [x] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/nf-core/ampliseq/tree/master/.github/CONTRIBUTING.md)
- [ ] If necessary, also make a PR on the nf-core/ampliseq _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
- [x] Make sure your code lints (`nf-core lint`).
- [x] Ensure the test suite passes (`nextflow run . -profile test,docker --outdir <OUTDIR>`).
- [ ] Check for unexpected warnings in debug mode (`nextflow run . -profile debug,test,docker --outdir <OUTDIR>`).
- [ ] Usage Documentation in `docs/usage.md` is updated.
- [ ] Output Documentation in `docs/output.md` is updated.
- [x] `CHANGELOG.md` is updated.
- [x] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

**Comment by d4straub** [comment]
Ah, almost everything is working again! What a journey!

**Review by erikrikarddaniel — APPROVED** [review]
Looks good!

**Review by mashehu — APPROVED** [review]

File: assets/multiqc_config.yml [hunk 1]
```diff
@@ -11,3 +11,9 @@ report_section_order:
     order: -1002
 
 export_plots: true
+
+disable_version_detection: true
+
+# MultiQC version 1.20 & 1.21 need that flag to produce plots!
+# TODO: Remove with version greater than 1.21!
```
[the remark this answers is not in the document]
Comment (d4straub) [reply assets/multiqc_config.yml:18]: This is a bug in multiqc, yes, I reported it and will be fixed in next release. I highly doubt that many use that folder as I do and care that its empty ;)

File: nextflow_schema.json [hunk 2]
```diff
@@ -21,13 +22,15 @@
                 },
                 "input_fasta": {
                     "type": "string",
+                    "format": "file-path",
                     "mimetype": "text/tsv",
                     "fa_icon": "fas fa-dna",
                     "description": "Path to ASV/OTU fasta file",
```
[the remark this answers is not in the document]
Comment (d4straub) [reply nextflow_schema.json:28]: No it doent need a pattern imho, but thanks for the suggestion!

File: nextflow_schema.json [hunk 3]
```diff
@@ -5,14 +5,15 @@
...
+        "input_output_options": {
             "title": "Main arguments",
             "type": "object",
             "description": "",
             "default": "",
             "properties": {
                 "input": {
                     "type": "string",
+                    "format": "file-path",
                     "mimetype": "text/tsv",
```
[the remark this answers is not in the document]
Comment (d4straub) [reply nextflow_schema.json:17]: No it doent need a pattern imho, but thanks for the suggestion!

File: nextflow_schema.json [hunk 4]
```diff
@@ -21,13 +22,15 @@
...
                     "fa_icon": "fas fa-dna",
                     "description": "Path to ASV/OTU fasta file",
                     "help_text": "Path to fasta format file with sequences that will be taxonomically classified. The fasta file input option can be used to taxonomically classify previously produced ASV/OTU sequences.\n\nThe fasta sequence header line may contain a description, that will be kept as part of the sequence name. However, tabs will be changed into spaces.\n\nRelated parameters are:\n- Choose an appropriate reference taxonomy for the type of amplicon (16S/18S/ITS/CO1) (default: DADA2 assignTaxonomy and 16S rRNA sequence database)"
                 },
                 "input_folder": {
                     "type": "string",
+                    "format": "directory-path",
                     "mimetype": "text/tsv",
```
[the remark this answers is not in the document]
Comment (d4straub) [reply nextflow_schema.json:34]: Good catch!

File: main.nf [hunk 5]
```diff
@@ -13,55 +13,75 @@ nextflow.enable.dsl = 2
...
-}
-
-// Validate input parameters
-if (params.validate_params) {
-    validateParameters()
-}
-
-WorkflowMain.initialise(workflow, params, log, args)
+include { AMPLISEQ  } from './workflows/ampliseq'
+include { PIPELINE_INITIALISATION } from './subworkflows/local/utils_nfcore_ampliseq_pipeline'
```
[the remark this answers is not in the document]
Comment (d4straub) [reply main.nf:21]: Indeed :)

**Comment by d4straub** [comment]
Thanks you two for your reviews, will merge it after tests pass.

**Pull request merged**